source/include/stdbool.readme

```diff
@@ -0,0 +1,31 @@
+
+#ifndef _STDBOOL_H
```

Comment:
Should there be a GitHub Action job to actually run the build check with the `stdbool.readme` and `stdint.readme` files to validate their definitions?

Reply:
Added. Couldn't think of a good name for it though